python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -995,6 +1008,81 @@ def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList
         if nImage is not None:
             nImage.assign(subNImage, bbox)
 
+    def assembleOnlineMeanCoadd(self, coaddExposure, tempExpRefList, imageScalerList, weightList,
+                                altMaskList, statsCtrl, nImage=None):
+        """Assemble the coadd using the "online" method.
+
+        This method takes a running sum of images and weights to save memory.
+        It only works for MEAN statistics.
+
+        Parameters
+        ----------
+        coaddExposure : `lsst.afw.image.Exposure`
+            The target exposure for the coadd.
+        tempExpRefList : `list`
```

Comment:
Could you add the explicit type of the object these variables hold (although they are not present in `assembleSubregion`)?
https://developer.lsst.io/python/numpydoc.html#sequence-types

Reply:
This is left blank in all the docstrings ... and it could be the gen2 or gen3 types ... but I can add something.

Reply:
Or maybe not.  This is a much bigger kettle of fish, it has multiple names, and the pain point is the gen2/gen3 overlap which will be cleaned out at some point.